.github/workflows/ci.yml

```diff
@@ -91,15 +91,14 @@ jobs:
     runs-on: [self-hosted, pyfluent]
     strategy:
       matrix:
-        #image-tag: [latest, solver-latest]
-        image-tag: [latest]
+        image-tag: [latest, solver-latest]
```

Comment:
OK, so `latest` would just be the cad+solver image and `solver-latest` would be the solver only image.  So, staring at these changes what it would do is run the CI/CD 2x right?

Reply:
i.e. I think the `latest` image is constructed using this Dockerfile:

https://github.com/pyansys/pyfluent/blob/main/docker/fluent/Dockerfile.meshing

You can see on line 6 that it is adding the CAD content to the `solver-latest` image which is built using this file

https://github.com/pyansys/pyfluent/blob/main/docker/fluent/Dockerfile

Reply:
I think the second run with `solver-latest` would (or should) not actually be able to build the documentation because that requires the cad import right.  The PR CI/CD with the `solver-latest` image did not rebuild the examples:

https://github.com/pyansys/pyfluent/runs/7433287000?check_suite_focus=true

I don't think it would have worked if this actually ran.

Reply:
Yeah, I misunderstood the tag names, we don't have the 23.1 image yet. I'll update the .yml to upload the artifacts for 22.2 image only.